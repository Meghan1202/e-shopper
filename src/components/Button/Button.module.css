.base {
  text-transform: uppercase;
  padding: 1rem;
  border-radius: 0.25rem;
  box-shadow: 0.125rem 0.125rem 0.25rem 0.125rem #d6d6d6;
  cursor: pointer;
}

.primary {
  font-size: 0.5rem;
  background-color: #fad873;
  padding: 1rem 2rem;
  border: 1px solid rgb(189, 189, 189);
}

.secondary {
  font-size: 1rem;
  background-color: #f5f5f5;
  color: rgb(82, 82, 82);
  font-weight: 400;
  padding: 1rem 2rem;
  border: 1px solid rgb(189, 189, 189);
}
