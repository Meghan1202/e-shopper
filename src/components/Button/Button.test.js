import { render } from '@testing-library/react';
import React from 'react';
import Button from './Button';

describe(Button.name, () => {
  test('Should match the snapshot', () => {
    // eslint-disable-next-line react/no-children-prop
    const { container } = render(<Button className="abc" children="xyz" onClick={jest.fn()} />);
    expect(container).toMatchSnapshot();
  });
});
