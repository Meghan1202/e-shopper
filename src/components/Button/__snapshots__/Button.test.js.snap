// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Should match the snapshot 1`] = `
<div>
  <button
    class="abc secondary"
    type="button"
  >
    xyz
  </button>
</div>
`;
